.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -0,0 +1,66 @@
+
+#
+# Checks if QA approvals are necessary, and if so runs the associated action when a new review is added
+#
+
+name: '[Utils] QA required approval'
+
+on:
+  pull_request:
+    types: [opened, edited, converted_to_draft, ready_for_review, reopened]
```

Comment:
What happens if QA approves before engineering? Should we clear off the QA approval?

Reply:
It will pass with a message 'this workflow has QA approval'

Reply:
@IsraelleHub I don't think this is the desired outcome. Typically we want 1 engineer to have approved the PR first before QA takes a look, so if we can add logic for it to only pass if there has also been 1 approval from @department-of-veterans-affairs/flagship-mobile-reviewers that would be ideal.

Reply:
Done. @narin please review